internal/e2e/logs.go

```diff
@@ -0,0 +1,94 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+// PrintDaemonSetPodLogs retrieves logs from each container in each pod of a DaemonSet.
+// namespace & labelSelector identify the DaemonSet's pods (e.g. "default", "app=containerd-check").
+func PrintDaemonSetPodLogs(
+	ctx context.Context,
+	restConfig *rest.Config,
+	namespace string,
+	labelSelector string,
+	t *testing.T,
```

Comment:
```suggestion
func PrintDaemonSetPodLogs(
	t *testing.T,
	ctx context.Context,
	restConfig *rest.Config,
	namespace string,
	labelSelector string,
```

IMO conventional ordering on arguments. t, ctx, ..args